test-cases/features/mixed-shard-bootstrap.yaml

```diff
@@ -0,0 +1,20 @@
+test_duration: 1200
+
+n_db_nodes: 2
```

Comment:
Is it intentional to have here only 2 nodes?

The smp list is for 3 nodes and it changes the third node that doesn't exist...

Reply:
Yes, the third node is added during the test and it gets the value from the list